cmake/HandlePrintConfiguration.cmake

```diff
@@ -29,6 +29,8 @@ if(NOT MSVC AND NOT XCODE_VERSION)
     print_config("C++ compilation flags" "${CMAKE_CXX_FLAGS} ${CMAKE_CXX_FLAGS_${CMAKE_BUILD_TYPE_UPPER}}")
 endif()
 
+print_config("Enable Boost serialization" "${GTSAM_ENABLE_BOOST_SERIALIZATION}")
```

Comment:
Printing the boost serialization flag to the console made sense to me. This is just a print operation.